docs/calculations.mdx

```diff
@@ -138,6 +140,7 @@ Product carousel
 | `network_type`                       | See [network types](#network-types) |
 | `property`                           | See [property](#property) |
 | `creative_ad_format`                 | Either a basic or vendor-provided ad format |
+| `creative_ad_platforms`              | Ad serving, verification, and measurement ad platforms wrapping or embedded in the creative |
```

Comment:
Is this a list of platforms ? How do we know creative ad server vs video player owner ? 

Should this be the canonical domain?

Reply:
I'm not sure we need to define types in this repo - this is a reference to the ad platform object, and feels like implementation could decide how to map to it? I guess if we wanted to standardize the delivery row so it would work across implementations then you are right, should be the canonical domain.